website/docs/jellyfish/api/loan.md

```diff
@@ -217,6 +217,30 @@ interface UTXO {
 }
 ```
 
+## updateLoanToken
+
+Updates an existing loan token.
+
+```ts title="client.loan.updateLoanToken()"
+interface loan {
```

Comment:
```suggestion
class Loan {
```

Reply:
We should use `interface` here. See other functions in the other md files.